src/firefly/java/edu/caltech/ipac/firefly/server/packagedata/DownloadScriptWorker.java

```diff
@@ -118,7 +118,7 @@ public String doCommand(SrvParam params) throws Exception {
 
                     //retrieve attributes based on scriptType
                     List<ScriptAttributes> attribute = scriptTypeMap.getOrDefault(scriptType.toLowerCase(), text);
-                    DownloadScript.composeDownloadScript(outFile, "Euclid", urlList, attribute);
+                    DownloadScript.composeDownloadScript(outFile, dlreq.getDataSource(), urlList, attribute);
```

Comment:
Thank you for getting to this and generalizing it. I'd forgotten the hardcoded "Euclid" in there.

Reply:
No problem